sdk/textanalytics/ai-text-analytics/src/analyzeBatchActionsResult.ts

```diff
@@ -2,29 +2,137 @@
 // Licensed under the MIT license.
 
 import { PagedAsyncIterableIterator } from "@azure/core-paging";
-import { ExtractKeyPhrasesResultArray } from "./extractKeyPhrasesResultArray";
-import { TextDocumentBatchStatistics } from "./generated/models";
-import { RecognizeCategorizedEntitiesResultArray } from "./recognizeCategorizedEntitiesResultArray";
-import { RecognizePiiEntitiesResultArray } from "./recognizePiiEntitiesResultArray";
+import {
+  ExtractKeyPhrasesResultArray,
+  makeExtractKeyPhrasesResultArray
+} from "./extractKeyPhrasesResultArray";
+import {
+  AnalyzeJobState as GeneratedResponse,
+  TasksStateTasksEntityRecognitionPiiTasksItem,
+  TasksStateTasksEntityRecognitionTasksItem,
+  TasksStateTasksKeyPhraseExtractionTasksItem,
+  TextDocumentBatchStatistics,
+  TextDocumentInput
+} from "./generated/models";
+import {
+  makeRecognizeCategorizedEntitiesResultArray,
+  RecognizeCategorizedEntitiesResultArray
+} from "./recognizeCategorizedEntitiesResultArray";
+import {
+  makeRecognizePiiEntitiesResultArray,
+  RecognizePiiEntitiesResultArray
+} from "./recognizePiiEntitiesResultArray";
+import { ErrorCode, intoTextAnalyticsError, TextAnalyticsError } from "./textAnalyticsResult";
 
 /**
- * The results of a successful analyze batch actions operation.
+ * The results of a succeeded analyze batch actions operation.
```

Comment:
nit: i think now they don't have to be successful to be included in this array since we're adding errors to this array

Reply:
Good catch! Fixed.